src/components/NewLexemeForm.vue

```diff
@@ -46,20 +46,26 @@ export default {
 		<lexical-category-input
 			v-model="lexicalCategory"
 		/>
-		<wikit-button
-			class="form-button-submit"
-			type="progressive"
-			variant="primary"
-		>
-			{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
-		</wikit-button>
+		<div>
+			<wikit-button
+				class="form-button-submit"
+				type="progressive"
+				variant="primary"
+			>
+				{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
+			</wikit-button>
+		</div>
 	</form>
 </template>
 
 <style scoped lang="scss">
 @import "@wmde/wikit-tokens/variables";
 
 .wbl-snl-form {
+	& > *:not(:last-child) {
```

Comment:
The `*` here is redundant, right? (It also seems to be compiled away anyways.)

Reply:
Apparently it is :astonished: . Though it is not being compiled away for me. The CSS in the browser changes for me when the `*` is removed.